src/containers/ConceptPage/ConceptForm/ConceptForm.tsx

```diff
@@ -136,10 +136,6 @@ const ConceptForm = ({
   const { data: licenses = [] } = useLicenses({ placeholderData: [] });
   const { ndlaId } = useSession();
 
-  useEffect(() => {
-    setSavedToServer(false);
-  }, [concept]);
```

Comment:
Dette fikser at lagreknappen for forklaringer ikke forblir grønn etter lagring inntil noe er oppdatert, tror det blir riktig å fjerne helt, men gjerne se om dere kommer på noen tilfeller jeg har glemt der denne er nødvendig!